### PR TITLE
ISSUE-29: Fix broken dependencies after Frictionless Package update

### PR DESCRIPTION
**Pull request by DiegoPino**
See #29 

@pcambra @giancarlobi could you give this a quick sanity check? @giancarlobi can you test if SBR works?

I also removed some old garbage I left around during development, a not used Service (which was also not working), a controller and some access functions we are not using right now.

@pcambra I will also open an issue for general cleanup of this module. Its a bit messy (with a bit means maybe too much)

### CODE REVIEW

File: src/Plugin/StrawberryRunnersPostProcessorPluginBase.php [hunk 1]
```diff
@@ -194,6 +194,43 @@ protected function kill($pid) {
...
+  /**
+   * Replace the first occurrence of a given value in the string.
+   * @see https://github.com/phannaly/laravel-helpers/blob/v1.0.3/src/String.php#L308
+   *
+   * @param  string  $search
+   * @param  string  $replace
+   * @param  string  $subject
+   * @return string
+   */
+   public function str_replace_first(string $search, string $replace, string $subject) {
```
Comment (pcambra) [inline src/Plugin/StrawberryRunnersPostProcessorPluginBase.php:224]: I think we should keep the camelCase notation

File: src/Form/StrawberryRunnersToolsForm.php [hunk 2]
```diff
@@ -152,37 +150,8 @@ public function callJmesPathprocess(array &$form, FormStateInterface $form_state
...
-
-  /**
-   * Limit access to the Tools according to their restricted state.
-   *
-   * @param \Drupal\Core\Session\AccountInterface $account
-   *   The account object.
-   * @param int $node
-   *   The node id.
-   */
-  public function accessTools(AccountInterface $account, $node) {
```
Comment (pcambra) [inline src/Form/StrawberryRunnersToolsForm.php:168]: Just making sure that the remove is intentional

File: src/Plugin/StrawberryRunnersPostProcessorPluginBase.php (hunk 1 above)
Comment (DiegoPino) [reply src/Plugin/StrawberryRunnersPostProcessorPluginBase.php:224]: Make sense. Updating

File: src/Plugin/StrawberryRunnersPostProcessorPluginBase.php [hunk 3]
```diff
@@ -194,6 +194,43 @@ protected function kill($pid) {
...
+  /**
+   * Replace the last occurrence of a given value in the string.
+   * @see https://github.com/phannaly/laravel-helpers/blob/v1.0.3/src/String.php#L331
+   *
+   * @param  string  $search
+   * @param  string  $replace
+   * @param  string  $subject
+   * @return string
+   */
+  public function str_replace_last(string $search, string $replace, string $subject) {
```
Comment (pcambra) [inline src/Plugin/StrawberryRunnersPostProcessorPluginBase.php:206]: Do we use this at all?

File: src/Form/StrawberryRunnersToolsForm.php (hunk 2 above)
Comment (DiegoPino) [reply src/Form/StrawberryRunnersToolsForm.php:168]: Yes (its Monday so I may be mistaken?). Was not used (checked whole code base). Will refactor the route properly and rewrite it (was also using undefined classes so quite bad)

File: src/Plugin/StrawberryRunnersPostProcessorPluginBase.php (hunk 3 above)
Comment (DiegoPino) [reply src/Plugin/StrawberryRunnersPostProcessorPluginBase.php]: No, we don't but it is the matching function to the one we use. I see uses for that one soon

Comment (pcambra) [reply src/Plugin/StrawberryRunnersPostProcessorPluginBase.php]: I kinda liked the pattern from that library, maybe we could have a regular archipelago utils PHP library in the future with commonalities.

**Comment by giancarlobi** [comment]
@DiegoPino @pcambra test in production site worked well!!

**Pull request merged**